src/docs/Components/Field/code.mdx

```diff
@@ -0,0 +1,211 @@
+---
+title: 'Code'
+order: 2
+---
+
+## Import
+
+Import the settings, the text-input and the fields `scss` files.
+
+```scss
+@import 'settings-tools/_all-settings';
+
+@include import-font-families();
+
+@import 'components/_c.text-input';
+
+@import 'components/_c.fields';
+
+```
+
+## Basic usage
+
+To create a fields you have two main class to apply :
+- `mc-field` on container
+- `mc-field__label` on label
+
+and reuse the `mc-text-input` class on the input
+
+```html
+<div class="mc-field">
+
+  <div class="mc-field__label-wrapper">
+    <label class="mc-field__label" for="inp">
+      Label
+    </label>
+  </div>
+    
+  <input id="inp" type="text" class="mc-text-input" placeholder="Empty" name="example" />
+
+</div>
+```
+
+<Preview path="fields" />
+
+## Variations
+
+### Available sizes
+
+You can use any Variation of the `input-text` : [Text input component documentation](/Components/TextInput/)
```

Comment:
I think this text could be fleshed out a little bit to make it more understandable, with something like this: 

Variations sizes concern only the input field element. So you can use all variations of the text-input.
See the [Text input component documentation](/Components/TextInput/) for more information

Reply:
I would not talk about size at all. 
Only Input variations : 

### Input variations 

You can use any of the existing Input-text component variations, 

Read the Text input component documentation for more information